platform/windows/display_server_windows.cpp

```diff
@@ -2995,13 +2995,17 @@ DisplayServerWindows::~DisplayServerWindows() {
 		SetWindowLongPtr(windows[MAIN_WINDOW_ID].hWnd, GWLP_WNDPROC, (LONG_PTR)user_proc);
 	};
 
-	if (windows.has(MAIN_WINDOW_ID)) {
+	// Destroy all remaining windows in windows, even though they should already been destroyed,
```

Comment:
We should figure out why they're not being destroyed instead of working it around here. A crash is actually good if it's the sign of a bug that we need to fix.

Reply:
My Visual Studio has trouble showing me what's inside the Map therefor I can't really read which windows cause that, since I only have the callstack